extensions/amp-access/0.1/jwt.js

```diff
@@ -80,9 +136,27 @@ export class JwtHelper {
       header: tryParseJson(decodeBase64WebSafe(parts[0]), invalidToken),
       payload: tryParseJson(decodeBase64WebSafe(parts[1]), invalidToken),
       verifiable: `${parts[0]}.${parts[1]}`,
-      sig: decodeBase64WebSafe(parts[2]),
```

Comment:
So, two reasons:
 1. It's a minor but still unnecessary work in PROD where signature verification will not be performed.
 2. It bugged me to keep a bin-string around. The full decoding of signature is base64 + binary array buffer. I could do that, but then, see (1).